apps/condo/domains/acquiring/schema/ExportPaymentsService.js

```diff
@@ -93,8 +94,12 @@ const ExportPaymentsService = new GQLCustomSchema('ExportPaymentsService', {
                     return {
                         date: formatDate(obj.advancedAt),
                         account: obj.accountNumber,
-                        address: get(obj, ['receipt', 'property', 'address'], ''),
-                        unitName: get(obj, ['receipt', 'account', 'unitName'], ''),
+                        address: isInvoicePayments ?
```

Comment:
using giant ternary operators here makes it harder to read this code, please consider exporting it to variables